contracts/lib.rs

```diff
@@ -374,6 +374,29 @@ mod open_payroll {
             Ok(())
         }
 
+        /// Add a new multiplier
+        #[ink(message)]
+        pub fn add_multiplier(&mut self, name: String) -> Result<(), Error> {
+            self.ensure_owner()?;
+            if self.multipliers_list.len() + 1 > MAX_MULTIPLIERS {
+                return Err(Error::MaxMultipliersExceeded);
+            }
+
+            //check if all payments are up to date
+            // TODO: this is really necesary ?
+            self.ensure_all_claimed_in_period()?;
```

Comment:
I dont think this should be necessary, because it does not add any value to anyone.